packages/editor-ui/src/stores/folders.store.ts

```diff
@@ -0,0 +1,111 @@
+import { defineStore } from 'pinia';
+import { computed, ref } from 'vue';
+import { STORES } from '@/constants';
+import type { FolderCreateResponse, FolderShortInfo, FolderTreeResponseItem } from '@/Interface';
+import * as workflowsApi from '@/api/workflows';
+import { useRootStore } from './root.store';
+
+export const useFoldersStore = defineStore(STORES.FOLDERS, () => {
+	const rootStore = useRootStore();
+
+	const currentFolderId = ref<string | null>(null);
```

Comment:
Can you please explain why this is needed here in the store? It's only used in the workflowsview and reused there. 
And it's passed as query param to other views, which is great and how it should be.

Reply:
Now looking at it, we can remove the part where we initialize editor for new workflow and pass parent folder id because it's not available when the workflow is actually saved.

Reply:
Makes sense to keep it, just in the workflows view, as it's only used there and passed around using query params, which is great.

Reply:
Sorry if my response was not clear
I just think it should be just at the view level
not the global level, like it's now
So some change is still needed, unless you still disagree.